docs/00_introduction/release.md

```diff
@@ -6,17 +6,29 @@ On this page you will learn about the Garden Linux releases and their lifecycle.
+
+### Patch releases _("patches")_
+
+During the nine-month maintenance period, updates are provided as patch releases (e.g., "1443.1" or "patch-1443.1" or "patch 1 of release 1443"). "Patches" are considered stable, and adopters are encouraged to always use the latest patch release of the latest stable release. Patch releases can be expected to adhere to following rules:
+
+- Can include higher minor versions of software packages or libraries already included in the stable release.
+- Can include security enhancements related to vulnerabilities identified after the release of the previous patch.
+- Can remove in rare (exceptional) situations software packages or libraries (e.g. due to security or legal risk identified after the release was published.). Such changes are documented in the release note. 
 
 ### Nightly releases 
 
-Triggered every night via the GitHub Action [Nightly Build](https://github.com/gardenlinux/gardenlinux/actions/workflows/nightly.yml). Their status can be monitored publicly in the action. The maintainers are striving towards having every 24 hours a successful build. This release contains latest software packages, aiming at rapid iterations for advanced stakeholders interested in the cutting-edge changes, but is not guaranteed to be always stable and as maintained/tested as the LTS releases. This release does not include maintenance commitments and therefore is not recommended for production scenarios. Up to once per quarter a daily release is promoted to stable release and accepted for nine months of standard maintenance.
+Every night, a new build is triggered via the GitHub Action [Nightly Build](https://github.com/gardenlinux/gardenlinux/actions/workflows/nightly.yml) (e.g., "nightly-1660.0"). You can monitor their status publicly on the action page. The maintainers aim to have a successful build every 24 hours. Occasionally, there might be nightly patches (like "nightly-1660.1") released if the initial nightly build requires urgent fixes. Nightly releases contain the latest software packages and are geared toward rapid iterations for advanced stakeholders interested in cutting-edge changes. However, they are not guaranteed to be as stable or as thoroughly maintained and tested as the stable releases. Nightly releases do not include maintenance commitments and are therefore not recommended for production environments. Up to once per quarter, a nightly release is promoted to a stable release and is then supported with nine months of standard maintenance.
+
+### Next Release
+
+The "next" release is a projection of the upcoming stable release. It exists primarily to give users an idea of when the next stable version **might** be released and how long it **might** be supported.

```

Comment:
that's new to me, what is that? build from an experimental repo?

Reply:
This did exist in here already ...
  - https://github.com/gardenlinux/gardenlinux/blob/f27744b247fb05b858fbfcfa272196a871094168/docs/00_introduction/release.md?plain=1#L35-L37
  - https://github.com/gardenlinux/gardenlinux/blob/f27744b247fb05b858fbfcfa272196a871094168/docs/00_introduction/release.md?plain=1#L55

... and I basically just defined it.

Is "a projection of the upcoming stable release" not clear enough? We could name this a "virtual release" to make it clearer that it is not something that you can use.

Reply:
I understood it as 'there are artifacts with next-xy names' but that's not the case, right? Maybe something @pnpavlov can clarify? I'm fine with having that in there but it seems like something that does not really exist to me so maybe we can just get rid of it?

If it's useful then I'm also fine with keeping it in, I'm just not aware of the value.